editor/src/core/shared/math-utils.ts

```diff
@@ -741,24 +741,6 @@ export function lineIntersection<C extends CoordinateMarker>(
   } as Point<C>
 }
 
-// Returns which side of the line is point b. If looking from lineA to lineB it is on the left, it returns the value 'left', etc.
```

Comment:
why did you remove this? is it problematic?

Reply:
I removed this because it is not necessary for the new algorithm, and it was only wrote and used for this.